etc/config/hlsl.amazon.properties

```diff
@@ -15,3 +11,37 @@ compiler.dxc_1_6_2112.exe=/opt/compiler-explorer/dxc-1.6.2112/bin/dxc
 compiler.dxc_1_6_2112.semver=1.6.2112
 compiler.dxc_1_7_2207.exe=/opt/compiler-explorer/dxc-1.7.2207/bin/dxc
 compiler.dxc_1_7_2207.semver=1.7.2207
+
+group.rga.compilers=rga262_dxctrunk:rga262_dxc172207:rga262_dxc162112:rga261_dxc172207:rga261_dxc162112
+group.rga.groupName=RGA
+group.rga.isSemVer=true
+group.rga.compilerType=rga
+
+compiler.rga262_dxctrunk.exe=/opt/compiler-explorer/rga-2.6.2/rga
+compiler.rga262_dxctrunk.semver=2.6.2
+compiler.rga262_dxctrunk.dxcPath=/opt/compiler-explorer/dxc-trunk/bin/dxc
+compiler.rga262_dxctrunk.name=RGA 2.6.2 (DXC trunk)
+
+compiler.rga262_dxc172207.exe=/opt/compiler-explorer/rga-2.6.2/rga
```

Comment:
Sorry for the annoying combinatorics here. We don't have a good solution for co-dependent projects (we have this in a couple other places too but mostly we go with "latest Y at the time X was released" for this stuff. I'm not saying this should be changed, just noting this may get unwieldy pretty quickly.